deployment/cdk/opensearch-service-migration/README.md

```diff
@@ -57,12 +57,12 @@ This is the core required stack of this CDK which is responsible for deploying t
 #### Network Stack (OSServiceNetworkCDKStack-STAGE-REGION)
 This optional stack will be used when the Domain is configured to be placed inside a VPC and will contain resources related to the networking of this VPC such as Security Groups and Subnets. It has a dependency on the Domain stack.
 
-#### Historical Capture Stack (OSServiceHistoricalCDKStack-STAGE-REGION)
-This optional stack sets up a ECS cluster to host/run Fetch Migration tasks for historical data migration. It has dependencies on both the Domain and Network stacks.
-
 #### Migration Assistance Stack (OSServiceMigrationCDKStack-STAGE-REGION)
 This optional stack is used to house the migration assistance resources which are in the process of being developed to assist in migrating to an OpenSearch domain. It has dependencies on both the Domain and Network stacks.
 
+#### Fetch Migration Stack (OSServiceHistoricalCDKStack-STAGE-REGION)
+This optional stack sets up a ECS cluster to host/run Fetch Migration tasks for data backfill / historical data migration. It has dependencies on the Domain, Network and Migration Assistance stacks.
```

Comment:
nit: 
"an ECS cluster"

nit: inconsistency on spacing around '/' (host/run and backfill / historical)

Reply:
Fixed